installer/cli/microk8s.py

```diff
@@ -188,6 +191,64 @@ def uninstall() -> None:
     echo.info("Thank you for using MicroK8s!")
 
 
+def dashboard_proxy() -> None:
+    vm_provider_name = "multipass"
+    vm_provider_class = get_provider_for(vm_provider_name)
+    echo = Echo()
+    try:
+        vm_provider_class.ensure_provider()
+    except ProviderNotFound as provider_error:
+        if provider_error.prompt_installable:
+            if echo.is_tty_connected():
+                echo.warning("MicroK8s is not installed. Please run `microk8s install`.")
+            return 1
+        else:
+            raise provider_error
+
+    instance = vm_provider_class(echoer=echo)
+
+    echo.info("Checking if Dashboard is running.")
+    command = ["microk8s.enable", "dashboard"]
+    output = instance.run(command, hide_output=True)
+    if b"Addon dashboard is already enabled." not in output:
+        echo.info("Waiting for Dashboard to come up.")
+        command = ["microk8s.kubectl", "-n", "kube-system", "wait",
```

Comment:
I go this error:
```
jackal@aurora:~/workspace/microk8s/installer$ ./dist/microk8s dashboard-proxy
Checking if Dashboard is running.
Waiting for Dashboard to come up.
error: timed out waiting for the condition on deployments/kubernetes-dashboard
An error occurred when trying to execute 'sudo microk8s.kubectl -n kube-system wait deployment kubernetes-dashboard --for condition=available' with 'multipass': returned exit code 1.
``` 

I think we should add a `--timeout=240s` or something large enough to catch the case of a dashboard that will not come up within the default 30s of the waiting time.

Reply:
Added the timeout. Feel free to merge if you agree with it.